Makefile

```diff
@@ -52,42 +51,45 @@ generate-from-swagger:
 		quay.io/goswagger/swagger:v0.25.0 generate client	--template=stratoscale -f swagger.yaml \
 		--template-dir=/templates/contrib
 
+generate-python-client: $(BUILD_FOLDER)
+	rm -rf $(BUILD_FOLDER)/assisted-service-client*
+	docker run --rm -u ${UID} --entrypoint /bin/sh \
+		-v $(BUILD_FOLDER):/local:Z \
+		-v $(ROOT_DIR)/swagger.yaml:/swagger.yaml:ro,Z \
+		-v $(ROOT_DIR)/tools/generate_python_client.sh:/script.sh:ro,Z \
+		-e SWAGGER_FILE=/swagger.yaml -e OUTPUT=/local/assisted-service-client/ \
+		swaggerapi/swagger-codegen-cli:2.4.15 /script.sh
+	cd $(BUILD_FOLDER)/assisted-service-client/ && python3 setup.py sdist --dist-dir $(BUILD_FOLDER)
+
+generate-keys:
+	cd tools && go run auth_keys_generator.go -keys-dir=$(BUILD_FOLDER)
+
+##################
+# Build & Update #
+##################
+
+.PHONY: build
+build: lint unit-test build-minimal generate-keys
+
+build-minimal: $(BUILD_FOLDER)
+	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service cmd/main.go
+
 build-onprem: build
 	podman build -f Dockerfile.assisted-service-onprem -t ${SERVICE} .
 
-##########
-# Update #
-##########
-
-build-image: build create-python-client
+build-image: build
 	GIT_REVISION=${GIT_REVISION} docker build --build-arg GIT_REVISION -f Dockerfile.assisted-service . -t $(SERVICE)
 
 update: build-image
 	docker push $(SERVICE)
 
-update-minimal: build-minimal create-python-client
+update-minimal: build-minimal
 	GIT_REVISION=${GIT_REVISION} docker build --build-arg GIT_REVISION -f Dockerfile.assisted-service . -t $(SERVICE)
 
-update-minikube: build create-python-client
+update-minikube: build
```

Comment:
We don't need the client anymore? test-infra is not using it?

Reply:
we need it but it is generated by the Dockerfile itself